source/benchmark.rst

```diff
@@ -0,0 +1,104 @@
+Benchmarking your Pulsar setup
+==============================
+
+To determine the performance of your Pulsar setup, you can use the
+`GalaxyBenchmarker <https://github.com/AndreasSko/Galaxy-Benchmarker>`_.
+This is a framework that allows for easy benchmarking of various Galaxy
+job destinations.
+
```

Comment:
Could you add a prerequisites section describing the needs to have Docker Compose and pointing to Docker Compose overview (https://docs.docker.com/compose/)?